service/create/service.go

```diff
@@ -94,24 +94,40 @@ func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
 		s.Config.Logger.Log("info", "Booted cert-operator")
 
-		s.Config.Logger.Log("info", "Test issuing a cert")
-
 		cert := CertificateSpec{
-			ClusterID:        "cert-test",
-			CommonName:       "api.cert-test.giantswarm.io",
-			IPSANs:           []string{"10.0.0.4", "10.0.0.5"},
-			AltNames:         []string{"api.k8s.cert-test.giantswarm.io"},
-			AllowBareDomains: false,
+			ClusterID:  "cert-test",
+			CommonName: "api.cert-test.g8s.eu-west-1.aws.test.private.giantswarm.io",
+			IPSANs:     []string{"10.0.0.4", "10.0.0.5"},
+			AltNames: []string{
+				"kubernetes",
+				"kubernetes.default",
+				"kubernetes.default.svc",
+				"kubernetes.default.svc.cluster.local",
+			},
+			AllowBareDomains: true,
 			TTL:              "720h",
 		}
 
-		issueResp, err := s.Issue(cert)
+		// Ensure a PKI backend exists for the cluster.
```

Comment:
```
if err := s.setupPKIBackend(cert); err != nil {
   s.Config.Logger.Log("error", fmt.Sprintf("could not setup pki backend '%#v'", err))
   return
}

if err := s.setupPKIPolicy(cert); err != nil {
   s.Config.Logger.Log("error", fmt.Sprintf("could not setup pki policy '%#v'", err))
   return
}

issueResp, err := s.Issue(cert)
if err != nil {
   s.Config.Logger.Log("error", fmt.Sprintf("could not issue cert '%#v'", err))
   return
}
s.Config.Logger.Log("info", fmt.Sprintf("cert issue %s %s", cert.CommonName, issueResp.SerialNumber))
```

does this make sense (it's untested :D) ? i'd say this is more idiomatic. up for discussion, though